CHANGES.md

```diff
@@ -39,6 +39,8 @@ v8.0.0 unreleased
 - Expose `Dockerfile_distro.win10_docker_base_image` and
   `Dockerfile_distro.win10_base_tag` to get the Windows container base
   image and tags. (@MisterDA #63)
+- Support BuildKit 1.4 syntax of here-documents in `COPY` instructions. (@MisterDA #99)
+- Support BuildKit 1.4 `--link` flag in `ADD` and `COPY` instructions. (@MisterDA #99)
```

Comment:
This will need to move to `0.8.1 unreleased` depending on how impactful the breaking change is.